float8_experimental/float8_dynamic_linear.py

```diff
@@ -88,8 +88,12 @@ def from_float(cls, mod, emulate: bool = False) -> "Float8DynamicLinear":
                 "bias": False,
             }
             new_mod = cls(**super_kwargs)
-        new_mod.forward_config = ScaledMMConfig(emulate, not bool(emulate))
-        new_mod.backward_config = ScaledMMConfig(emulate, False)
+        new_mod.forward_config = ScaledMMConfig(
+            emulate, not bool(emulate), False, config.pad_inner_dim
```

Comment:
optional nit: since number of arguments is growing, could make more readable by doing `ScaledMM(arg_name=arg_val, ...)` instead of `ScaledMM(arg_val, ...)`, so reader doesn't need to go back to `ScaledMM` def to understand the initialization

Reply:
yeah, agreed will change